weave/integrations/bedrock/bedrock_sdk.py

```diff
@@ -0,0 +1,151 @@
+from typing import TYPE_CHECKING, Any, Optional, Callable
+
+import weave
+from weave.trace.weave_client import Call
+from weave.trace.op_extensions.accumulator import add_accumulator, _IteratorWrapper
+from weave.trace.patcher import MultiPatcher, SymbolPatcher
+
+if TYPE_CHECKING:
+    from botocore.client import BaseClient
+
+
+def bedrock_on_finish_converse(
+    call: Call, output: Any, exception: Optional[BaseException]
+) -> None:
+    model_name = call.inputs["modelId"]
+    usage = {model_name: {"requests": 1}}
+    summary_update = {"usage": usage}
+    if output:
+        tokens_metrics = {
+            "prompt_tokens": output["usage"]["inputTokens"],
+            "completion_tokens": output["usage"]["outputTokens"],
+            "total_tokens": output["usage"]["totalTokens"],
+        }
+        usage[model_name].update(tokens_metrics)
+    if call.summary is not None:
+        call.summary.update(summary_update)
+
+
+def bedrock_on_finish_invoke(
+    call: Call, output: Any, exception: Optional[BaseException]
```

Comment:
is output Any?  Seems like an optional dict from how you're using it

Reply:
same